app/src/main/java/com/compscieddy/meetinthemiddle/util/Util.java

```diff
@@ -63,4 +68,15 @@ public static void rotateLocationInactive(View view) {
         .start();
   }
 
+  public static Date getCurrentUTCTime(){
+    // Set device's time zone to UTC
+    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    Calendar calendar = Calendar.getInstance();
+    // Reset device's time zone to device local time zone
```

Comment:
This comment seems unrelated to the code...where is it begin reset? (maybe I just don't see it)